commands/dequeuer/main.go

```diff
@@ -69,19 +70,23 @@ func main() {
 	signal.Notify(sigterm, syscall.SIGTERM)
 	sig := <-sigterm
 	fmt.Printf("Caught signal %v, shutting down...\n", sig)
-	var wg sync.WaitGroup
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
This is just giving each of the groups 10 seconds to shut down gracefully, yeah?

Reply:
yes